src/Scripts/sitegen-theme-marker/sitegen-theme-marker.js

```diff
@@ -0,0 +1,21 @@
+window.onload = function () {
+	const homepages = window.nfdOnboarding.homepages.data;
```

Comment:
Should we add a `?` in case the user comes to the themes page without completing Onboarding?

Reply:
This will always be in an empty array from the beginning. In the backend, we have a backup check implemented as well `( ! $flow_data || ! isset( $flow_data['sitegen']['homepages'] ) )`